### ISSUE TITLE
Is it possible to set attributes for the tag <title /> ?

### ISSUE THREAD

**Issue by mirexdoors** [nuxt/vue-meta#732]
Hello!

According to the [HTML specification](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element), the global attributes are valid for the title tag. But I didn't see an api for this. Is it possible?

**Comment by the-m-git** [comment]
Тег title, насколько мне известно, ничем не отличается принципиально от тега div или любого другого, ему можно давать аттрибуты и даже классы, если сильно захочется. Просто у него display:none прописан в стандартные стили браузера. На сыром js можно куериСелектором выловить его и делать с ним всё, что захочется

(в стилях юсерАгента, пардон)

**Comment by the-m-git** [comment]
пруф (я отловил тег тайтл и повесил на него в консоли стили, ну и хед сделал блочным)
![image](https://user-images.githubusercontent.com/63388476/161527524-6f6bc3df-5b92-4efb-9a0a-c9c9036fe30a.png)

**Comment by mirexdoors** [comment]
> Тег title, насколько мне известно, ничем не отличается принципиально от тега div или любого другого, ему можно давать аттрибуты и даже классы, если сильно захочется. Просто у него display:none прописан в стандартные стили браузера. На сыром js можно куериСелектором выловить его и делать с ним всё, что захочется
> 
> (в стилях юсерАгента, пардон)

Да, спасибо, с этим понятно. Я там в вопросе не сформулировал, но имел ввиду, что хотелось бы иметь возможность делать это на стороне накста, а не на клиенте. То есть, чтобы это уже было в документе в момент загрузки (сейчас же для  <title /> можно задать только контент
